dice.js

```diff
@@ -0,0 +1,11 @@
+//Q2.Dice Roller.
+function rollDice(){
+let nums = [1, 2, 3, 4, 5, 6];
+let num1 = nums[Math.floor(Math.random() * nums.length)];
+let num2 = nums[Math.floor(Math.random() * nums.length)];
```

Comment:
You're already using Math.random() to get random numbers from 1 to 6, you don't need an array of numbers from 1 to 6.
`let num1 = Math.floor(Math.random() * 6) + 1 ;`
works without needing an array